Dockerfile

```diff
@@ -7,4 +7,5 @@ RUN apt update
 RUN apt install -y bash curl wget
 
 WORKDIR /
+COPY etcd_bootstrap_script.sh /var/etcd/bin/bootstrap.sh
 ENTRYPOINT ["/usr/local/bin/etcd"]
```

Comment:
Does it make sense to change the entrypoint to the bootstrap script?

Reply:
Since we pass the command manually in sts [here](https://github.com/gardener/etcd-druid/blob/ba48ea629e59585d1b71060c9fd91c8778a60ad6/charts/etcd/templates/etcd-statefulset.yaml#L53), I didn't feel the need to change the default entrypoint, since the etcd doesn't necessarily have to run as bootstrap script, but can still run etcd as a standalone etcd image